components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -285,6 +285,9 @@
 <mam4_mode2_file rad="rrtmg">atm/cam/physprops/mam4_mode2_rrtmg_c130628.nc</mam4_mode2_file>
 <mam4_mode3_file rad="rrtmg">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc</mam4_mode3_file>
 <mam4_mode4_file rad="rrtmg">atm/cam/physprops/mam4_mode4_rrtmg_c130628.nc</mam4_mode4_file>
+<!-- for CLDERA stratospheric aerosol treatment-->
```

Comment:
hardcoded to SNL HPC, problematic for NERSC.

Reply:
Yeah, I think the better thing to do here is to leave this unset, or else put in the relative path and know that we will need to configure this in the runscript if we do not move these datasets into the e3sm inputdata paths. I.e., I think I'd prefer to see here something like:
```
<mam4_mode1_file rad="rrtmg" cldera_strat_volc="1">atm/cam/physprops/modal_optics_mode1_hb.nc</mam4_mode1_file>
<mam4_mode3_file rad="rrtmg" cldera_strat_volc="1">atm/cam/physprops/modal_optics_mode3_hb.nc</mam4_mode3_file>
```